pkg/cvo/cvo.go

```diff
@@ -159,6 +162,8 @@ func New(
 	overridePayloadDir string,
 	minimumInterval time.Duration,
 	cvInformer configinformersv1.ClusterVersionInformer,
+	etcdInformer informers.GenericInformer,
+	etcdClient v1helpers.StaticPodOperatorClient,
```

Comment:
We're just looking at etcd's ClusterOperator, right?  I'd expected us to be able to use the existing `coLister` / `coInformer`, etc., without having to add a new informer/client here.

Reply:
Maru said they were updating conditions in "the resource will be apiVersion: operator.openshift.io/v1 kind: Etcd name: cluster"
`etcdInformer` is no longer needed I just hasn't removed it yet.